cli/flags.rs

```diff
@@ -263,6 +264,8 @@ static ENV_VARIABLES_HELP: &str = r#"ENVIRONMENT VARIABLES:
                          hostnames to use when fetching remote modules from
                          private repositories
                          (e.g. "abcde12345@deno.land;54321edcba@github.com")
+    DENO_TLS_CA_STORE    Comma-seperated list of certificate stores
+                         (defaults to "system,mozilla")
```

Comment:
Defaulting to both should be a seperate change, and is definitely not in scope for a `fix:`. Changing the defaults would have to wait for a minor (possibly a major).

Reply:
ok

Reply:
I changed the default to only be "mozilla". Loading from the keystore is non-default.